src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +100,16 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        if (someDate.isBefore(LocalDate.now())) {
+            return someDate + " is before " + LocalDate.now();
+        }
+        if (someDate.isAfter(LocalDate.now())) {
+            return someDate + " is after " + LocalDate.now();
+        }
+        if (someDate.isEqual(LocalDate.now())) {
```

Comment:
if (someDate.isEqual(LocalDate.now())) {
            return someDate + " is today";
        }
        return null;
-->
return someDate + " is today";